1-Draft/RFCNNNN-ConvertFrom-Html.md

```diff
@@ -0,0 +1,185 @@
+---
+RFC: RFC
+Author: Josh Rickard
+Status: Draft
+SupercededBy: 
+Version: 1.0
+Area: PowerShell Core Web CmdLets
+Comments Due: August 31st, 2018
+Plan to implement: 
+---
+
+# ConvertFrom-Html
+
+Currently the PowerShell Core Web CmdLets do not have access to the `HtmlWebResponseObject` and currently only contains the `BasicHtmlWebResponseObject` type.  Because of this, the capability to parse HTML using the `ParsedHtml` property of the `HtmlWebResponseObject` type does not exist within PowerShell Core.
+
+Windows PowerShell does contain the `HtmlWebResponseObject`, but PowerShell Core currently only contains the `BasicHtmlWebResponseObject` type.
+
+Additionally, Windows PowerShell Web CmdLets utilize Internet Explorer to parse HTML content. Since non-Windows systems do not have Internet Explorer, PowerShell Core utilizes the `BasicHtmlWebResponseObject` which does not contain this property.
+
+```text
+TypeName: Microsoft.PowerShell.Commands.BasicHtmlWebResponseObject
+
+Name              MemberType Definition
+----              ---------- ----------
+Equals            Method     bool Equals(System.Object obj)
+GetHashCode       Method     int GetHashCode()
+GetType           Method     type GetType()
+ToString          Method     string ToString()
+BaseResponse      Property   System.Net.Http.HttpResponseMessage BaseResponse {get;set;}
+Content           Property   string Content {get;}
+Encoding          Property   System.Text.Encoding Encoding {get;}
+Headers           Property   System.Collections.Generic.Dictionary[string,System.Collections.Generic.IEnumerable[stri...
+Images            Property   Microsoft.PowerShell.Commands.WebCmdletElementCollection Images {get;}
+InputFields       Property   Microsoft.PowerShell.Commands.WebCmdletElementCollection InputFields {get;}
+Links             Property   Microsoft.PowerShell.Commands.WebCmdletElementCollection Links {get;}
+RawContent        Property   string RawContent {get;set;}
+RawContentLength  Property   long RawContentLength {get;}
+RawContentStream  Property   System.IO.MemoryStream RawContentStream {get;}
+RelationLink      Property   System.Collections.Generic.Dictionary[string,string] RelationLink {get;}
+StatusCode        Property   int StatusCode {get;}
+StatusDescription Property   string StatusDescription {get;}
+```
+
+This RFC proposes that the creation of a new CmdLet named `ConvertFrom-Html`.  This CmdLet is to be implemented into PowerShell Core and should utilize the [AngelSharp](https://github.com/AngleSharp/AngleSharp) framework for converting strings into a PSCustomObject.
```

Comment:
Please add sub-title "Proposition".

"for converting strings into ..." -> "for converting HTML strings into ...".

Here and below - what is output object type or types? We mention - "objects", "PSCustomObject" ...

Reply:
Based on the RFC guidelines, i'm not sure if this is needed.